### PR TITLE
[NL] Add a an extra sentences for light_HassTurn(On | Off)

### PR DESCRIPTION
**Pull request by berendhaan**
I've seen this sentence come across the community (lamp [entity] aan), so I've added it to the list.

### CODE REVIEW

File: sentences/nl/light_HassTurnOn.yaml [hunk 1]
```diff
@@ -9,6 +9,8 @@ intents:
           - "Schakel [alle] <lamp> [in] <area> in"
           - "<doe> (alle | in) <area> <lamp> aan"
           - "Schakel (alle | in) <area> <lamp> in"
+          - "<lamp> aan"
```
Comment (TheFes) [inline sentences/nl/light_HassTurnOn.yaml:12]: this sentence is missing an <area> definition
It is not allowed to add sentences to turn on all lights, regardless of an area

File: sentences/nl/light_HassTurnOn.yaml [hunk 2]
```diff
@@ -9,6 +9,8 @@ intents:
...
+          - "<lamp> aan in <area>"
```
Comment (TheFes) [inline sentences/nl/light_HassTurnOn.yaml:13]: You can make `<doe>` optional in `"<doe> [alle] <lamp> [<naar>] aan in <area>"` to make this option possible without adding an additional sentence.

File: sentences/nl/light_HassTurnOff.yaml [hunk 3]
```diff
@@ -6,13 +6,15 @@ intents:
...
           - "<doe> (alle | in) <area> <lamp> [<naar>] uit"
+          - "<lamp> uit in <area>"
         response: lights_area
         slots:
           domain: light
           name: all
       - sentences:
           - "<doe> (overal | alle) <lamp> uit"
           - "<doe> <lamp> overal uit"
+          - "<lamp> uit"
```
Comment (TheFes) [inline sentences/nl/light_HassTurnOff.yaml:17]: this sentence is missing a test

File: tests/nl/light_HassTurnOn.yaml [hunk 4]
```diff
@@ -15,6 +15,8 @@ tests:
       - "Doe in de keuken de lampen aan"
       - "Mogen in de keuken de lampen aan?"
       - "Zet in de keuken het licht aan"
+      - "Lamp keukenlamp aan"
```
Comment (TheFes) [inline tests/nl/light_HassTurnOn.yaml:18]: This sentence doesn't match with any of the sentences.
Note that spaces are taken into account as of Hassil 1.0

This sentence would match with `<lamp> <area><lamp> aan` or with `<lamp> <name> aan`
Both do not match with any of the intents.

**Comment by TheFes** [comment]
@berendhaan I resolved the conflicts which were caused because a PR of mine was merged

File: sentences/nl/light_HassTurnOn.yaml (hunk 2 above)
Comment (berendhaan) [reply sentences/nl/light_HassTurnOn.yaml]: Makes sense, I've changed that.

**Comment by TheFes** [comment]
Looks fine now, thanks for your contribution @berendhaan

**Pull request merged**